geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/MessageDispatcher.java

```diff
@@ -83,7 +84,7 @@ public class MessageDispatcher extends LoggingThread {
   /**
    * Default value in milliseconds for waiting for re-authentication
    */
-  private static final long DEFAULT_RE_AUTHENTICATE_WAIT_TIME = 5000;
+  private static final long DEFAULT_RE_AUTHENTICATE_WAIT_TIME = Duration.of(1, MINUTES).toMillis();
```

Comment:
All of our other code uses `java.util.concurrent.TimeUnit` and/or `Duration.ofMinutes(1)` instead of using `java.time.temporal.ChronoUnit`. I think you should drop `ChronoUnit`.